packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -183,6 +183,63 @@ describe('RSTTraversal', () => {
       expect(spy.callCount).to.equal(4);
     });
 
+    it('should handle array children', () => {
+      const spy = sinon.spy();
+      const twoDivArray = [
+        <div key="a" />,
+        <div key="b" />,
+      ];
+      const divA = $(<div key="a" />);
+      const divB = $(<div key="b" />);
+      const node = $(
+        <div>
+          {twoDivArray}
```

Comment:
Suggest we add tests for sibling children (nested arrays and also nested iterables) e.g.

```
<div>
  {array1}
  {array2}
</div>
```

```
<div>
  {iterable1}
  {iterable2}
</div>
```